lib/cartopy/mpl/patch.py

```diff
@@ -249,3 +265,183 @@ def not_zero_poly(geom):
     result = list(filter(not_zero_poly, geom_collection))
 
     return result + other_result_geoms
+
+
+def path_to_shapely(path, force_ccw=False):
+    """
+    Create a Shapely geometric objects from a :class:`matplotlib.path.Path`.
+
+    Parameters
+    ----------
+    path
+        A :class:`matplotlib.path.Path` instance.
+
+    Other Parameters
+    ----------------
+    force_ccw
```

Comment:
Are we able to remove this parameter? I couldn't find any uses of `force_ccw` in Cartopy.